sdk/netapp/Azure.ResourceManager.NetApp/tests/NetAppManagementTestBase.cs

```diff
@@ -12,14 +12,19 @@ namespace Azure.ResourceManager.NetApp.Tests
 {
     public class NetAppManagementTestBase : ManagementRecordedTestBase<NetAppManagementTestEnvironment>
     {
-        protected ArmClient Client { get; private set; }
+        protected ArmClient Client { get; set; }
```

Comment:
I did not see a place that we are setting the Client
And actually we should not set the Client, because in our test framework, the ArmClient instance is instrumented to support various of validations and playback of the test cases, we should not replace the instance of ArmClient in any test cases.
Please add the private accessor back so that we could ensure only in this class, we will set the reference of `Client`.